modules/rostests/winetests/msi/CMakeLists.txt

```diff
@@ -23,6 +23,10 @@ list(APPEND SOURCE
     suminfo.c
     precomp.h)
 
+if(VS_SOLUTION)
+    # Let rc.exe find custom.dll in its subdirectory, i.e. Debug.
+    include_directories($<TARGET_FILE_DIR:custom>)
```

Comment:
Does anything break if you just do this unconditionally?

Reply:
According to other CMake files in the repo, include_directories should be up between add_definitions and spec2def

Reply:
Moved up.

I haven't tested this "unconditionally" with other build setups, yet.
While I assume it shouldn't harm, it seems useless to me to enable it when not needed...